### PR TITLE
style(Checkbox): update checkboxes

### PR DESCRIPTION
**Pull request by rapahaeru**
## Description
https://jirasoftware.catho.com.br/browse/QTM-63

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/Checkbox/Checkbox.jsx [hunk 1]
```diff
@@ -8,13 +8,20 @@ import Icon from '../Icon';
 import CheckboxGroupContext from './CheckboxGroupContext';
 import { colors, spacing, baseFontSize } from '../shared/theme';
 
-const CHECKBOX_SIZE = '18px';
+const CHECKBOX_SIZE = '24px';
```
Comment (pedrohmorais) [inline components/Checkbox/Checkbox.jsx]: What did you think about to take this value from `theme` ?

**Comment by pedrohmorais** [comment]
Error label needs more top spacing.

Quantum:
image

Styleguide:
image

File: components/Checkbox/Checkbox.jsx (hunk 1 above)
Comment (rapahaeru) [reply components/Checkbox/Checkbox.jsx]: I would keep it the way it is. Because this constant is used only here.
What do you think?

Comment (pedrohmorais) [reply components/Checkbox/Checkbox.jsx]: Ok

**Pull request merged**